docs/install/knative-with-operators.md

```diff
@@ -94,9 +120,60 @@ kind: KnativeServing
 metadata:
   name: knative-serving
   namespace: knative-serving
-EOF
+spec:
+  version: {{spec.version}}
+  manifests:
+    - URL: https://github.com/knative/serving/releases/download/v${VERSION}/serving-core.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-hpa.yaml
+    - URL: https://github.com/knative/serving/releases/download/v{VERSION}/serving-post-install-jobs.yaml
+    - URL: https://github.com/knative/net-istio/releases/download/v0.17.0/net-istio.yaml
 ```
 
+The field `spec.version` is used to set the version of Knative Serving. Replace `{{spec.version}}` with the correct version number.
+The tag `${VERSION}` is automatically replaced with the version number from `spec.version` by the operator.
+
+The field `spec.manifests` is used to specify one or multiple URL links of Knative Serving component. Do not forget to
+add the valid URL of the Knative network ingress plugin. Knative Serving component is still tightly-coupled with a network
+ingress plugin in the operator. As in the above example, you can use `net-istio`. The ordering of the URLs is critical.
```

Comment:
It sounds like uses can choose other ingress plugin. 
But only `net-istio` is supported now? (I assumed it because `Before you begin` section says `Download and Istio`.

Reply:
that's tricky. since we can specify the plugin, we technically can install any of them with the customized yaml.

Reply:
We don't have official support for non-Istio installs yet, but bring-your-own YAML provides an out.

Reply:
I understand. Thank you.